packages/headless/src/controllers/core/folded-result-list/headless-core-folded-result-list.test.ts

```diff
@@ -108,14 +150,14 @@ describe('FoldedResultList', () => {
       ).toBeFalsy();
     });
 
-    it('dispatches analytics and folding/loadCollection when calling loadCollection', () => {
-      foldedResultList.loadCollection(foldedResultList.state.results[0]);
-      expect(engine.actions.pop()?.type).toEqual(
-        'analytics/folding/showMore/pending'
-      );
-      expect(engine.actions.pop()?.type).toEqual(
-        'folding/loadCollection/pending'
-      );
+    it('#loadCollection dispatches #loadCollection AND #logShowMoreFoldedResults analytics', () => {
```

Comment:
#loadCollection dispatches #loadCollection sounds a bit odd, I believe we can improve the description of this test.